gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=0.2.0
+version=0.2.0-SNAPSHOT
```

Comment:
Didnt we merge the post release PR? The version should be updated

Reply:
There was. Closed in favor of this PR. Updated in [2f1909f](https://github.com/ballerina-platform/module-ballerina-yaml/pull/9/commits/2f1909fe4cf21431ebb096406fb73e18e0c91d29)